### PR TITLE
adjusts user stories for consistency

### LINKED ISSUE #229: Update user stories for consistency
User stories need to be consistent with the code we are expecting the students to write.

### PR DESCRIPTION
**Pull request by tansaku**
closes #229

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,44 +25,36 @@ Task
 We have a request from a client to write the software to control the flow of planes at an airport. The planes can land and take off provided that the weather is sunny. Occasionally it may be stormy, in which case no planes can land or take off.  Here are the user stories that we worked out in collaboration with the client:
 
 ```
-As an air traffic controller
-So planes can land safely at my airport
-I would like to instruct a plane to land
+As an air traffic controller 
+So I can get passengers to a destination 
+I want to instruct a plane to land at an airport and confirm that it has landed
```
Comment (sjmog) [inline README.md]: Separate user story?

`I want to instruct a plane to land at an airport` and `I want to see if a plane is landed or not`

File: README.md [hunk 2]
```diff
@@ -25,44 +25,36 @@ Task
...
+As an air traffic controller 
+So I can get passengers to a destination 
+I want to instruct a plane to land at an airport and confirm that it has landed 
 
-As an air traffic controller
-So planes can take off safely from my airport
-I would like to instruct a plane to take off
+As an air traffic controller 
+So I can get passengers on the way to their destination 
+I want to instruct a plane to take off from an airport and confirm that it is no longer in the airport
```
Comment (sjmog) [inline README.md]: Ditto to above

File: README.md [hunk 3]
```diff
@@ -25,44 +25,36 @@ Task
...
 
-As an air traffic controller
-So the system is consistent and correctly reports plane status and location
-I want to ensure a plane that is not flying cannot land and must be in an airport
 
-As an air traffic controller
-So the system is consistent and correctly reports plane status and location
-I want to ensure a plane that has taken off from an airport is no longer in that airport
-```
+Your code should defend against inconsistent states of the system ensuring that planes can only take off from airports they are in; planes that are already flying cannot takes off and/or be in an airport; planes that are landed cannot land again and must be in an airport.
```
Comment (sjmog) [inline README.md]: Mention 'edge cases' with a link to a description of the terminology?

**Comment by sjmog** [comment]
Much better than before :shipit:

File: README.md (hunk 1 above)
Comment (tansaku) [inline README.md]: I was combining these to try and make it simple to make a simple test that ensures the consequence, i.e. from my js version:

```javascript
    it('planes can be instructed to land at an airport', function(){
      plane.land(airport);
      expect(airport.planes()).toContain(plane);
    });
```

File: README.md (hunk 3 above)
Comment (tansaku) [inline README.md]: good call!  http://programmers.stackexchange.com/questions/125587/what-are-the-difference-between-an-edge-case-a-corner-case-a-base-case-and-a-b perhaps?

**Pull request merged**